dbhelper.py

```diff
@@ -1,5 +1,7 @@
 import logging
+import os
 import sqlite3
+from pathlib import Path
```

Comment:
Unused import, and please don't add any dependencies on pathlib (it doesn't do anything which Python couldn't do 20 years ago).

Reply:
As you say, code was in fork I adapted.